cross/arm-none-eabi-binutils/Portfile

```diff
@@ -3,9 +3,13 @@
 PortSystem          1.0
 PortGroup           crossbinutils 1.0
 
-crossbinutils.setup arm-none-eabi 2.33.1
+crossbinutils.setup arm-none-eabi 2.34
 maintainers         nomaintainer
 
+checksums           rmd160  e99f517407503cec1042a2e824b7c1f8f0fca4b8 \
+                    sha256  89f010078b6cf69c23c27897d686055ab89b198dddf819efb0a4f2c38a0b36e6 \
+                    size    32430470
```

Comment:
I'm curious how this port worked without checksums before?

Reply:
The checksums are stored in the portgroup.

See https://github.com/macports/macports-ports/blob/c9b0d4dd829df855e732348785cd0500cd92f766/_resources/port1.0/group/crossbinutils-1.0.tcl#L39

Reply:
Aha, that's where they are. Thanks!